cms/templates/js/embed-link-share-link-editor.underscore

```diff
@@ -0,0 +1,20 @@
+<div class="modal-section-content">
+    <div class="subsection-share-link-container">
+        <div>
+            <p>
+                <%- gettext('Embed your subsection content directly on a page using an iframe, and view it withput the LMS header and footer.') %>
+            </p>
+            <a
+                href="#"
+                title="<%- gettext('Click to copy link to clipboard') %>"
+                onclick="copyToClipboard('<%= embedLmsUrl %>')"
```

Comment:
Is there a way we can implement this attachment of the onclick event to this copyToClipboard function but in a backbone way? As I mentioned here:  https://github.com/openedx/edx-platform/pull/34043/files#r1450754846